liquibase-snowflake/src/main/java/liquibase/sqlgenerator/core/SetColumnRemarksGeneratorSnowflake.java

```diff
@@ -41,7 +41,7 @@ public ValidationErrors validate(SetColumnRemarksStatement statement, Database d
                 // Check if we're trying to set the column remarks on a view, and if so, note that this is not supported.
                 try {
                     List<Map<String, ?>> viewList = Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", database).queryForList(
-                            new RawSqlStatement(String.format("SHOW VIEWS LIKE '%s'", statement.getTableName())));
+                            new RawParameterizedSqlStatement("SHOW VIEWS LIKE '?'", statement.getTableName()));
```

Comment:
Same as before, I believe parameters won't work for internal commands such as `show` .